src/schema/ghga.yaml

```diff
@@ -278,15 +305,20 @@ classes:
       used and the Agent that is running the experiment.
     slots:
       - title
+      - name
```

Comment:
I think it is misleading to have both title and name. We should have either one or the other.

Reply:
Solved